examples/deployment-plus-min/nginx-ingress-controller.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: k8s.nginx.org/v1alpha1
+kind: NginxIngressController
+metadata:
+  name: my-nginx-ingress-controller
+  namespace: my-nginx-ingress
+spec:
+  type: deployment
+  nginxPlus: true
+  image:
+    repository: nginx/nginx-ingress
```

I suggest following the convention we have in the IC repo (https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/deployment/nginx-plus-ingress.yaml#L21). This will make it more clear that that field should be changed. Note that `nginx/nginx-ingress` exists on DockerHub, so if somebody deploys this manifests, the Kubernetes will be able to pull it
```suggestion
    repository: nginx-plus-ingress:edge
```